peps/pep-0725.rst

```diff
@@ -1,11 +1,13 @@
 PEP: 725
 Title: Specifying external dependencies in pyproject.toml
 Author: Pradyun Gedam <pradyunsg@gmail.com>,
+        Jaime Rodríguez-Guerra <jaime.rogue@gmail.com>,
         Ralf Gommers <ralf.gommers@gmail.com>
 Discussions-To: https://discuss.python.org/t/31888
 Status: Draft
 Type: Standards Track
 Topic: Packaging
+Requires: 804
```

Comment:
If this 725 requires 804, and [804](https://pep-previews--4572.org.readthedocs.build/pep-0804/) requires 725, do they both need to be approved as a single unit?

Reply:
We should probably drop `Requires` here, since we have a reference in the "Motivation" section already.

Reply:
Okay, will do. I only added it last-minute because Jaime asked about it for PEP 804, and it seems sensible to use it consistently between the two PEPs. Usage in other PEPs of this Requires field seems fairly rare and inconsistent anyway.